### ISSUE TITLE
warning Field is not callable

### ISSUE THREAD

**Issue by A-Iskakov** [typeddjango/django-stubs#47]
Hello guys,
I get multiple warnings while determine field in Model class, such as **EmailField, BooleanField, PositiveSmallIntegerField, PositiveIntegerField, DateTimeField, TextField, NullBooleanField, UUIDField, URLField**
The warning everywhere the same:
This inspection highlights attempts to call objects which are not callable, like, for example, tuples.

**Comment by mkurnikov** [comment]
I have this in PyCharm, is it present in mypy too?

**Comment by A-Iskakov** [comment]
I am also having this in Pycharm only

**Comment by syastrov** [comment]
I've also experienced this in PyCharm.

**Comment by raysry** [comment]
I've also experienced this in PyCharm!

**Comment by mkurnikov** [comment]
Difference between "callable" and "non-callable" fields in this case is that callable ones have their `__init__` method redefined, I assume bug in PyCharm is
(if this is a contents of stub file)
```
_T = TypeVar('_T')

class ParentField(Generic[_T]):
    def __init__(self):
        pass

class ChildField(ParentField[_T]):
    pass
```

`ChildField` will not be callable in source code, according to pycharm.

**Comment by IMBlues** [comment]
I've also experienced this in PyCharm.

**Comment by Rocamonde** [comment]
I am also experiencing this issue in PyCharm 2018.3.3, Python 3.7, Django 2.1

**Comment by A-Iskakov** [comment]
The problem almost fixed in Pycharm 2019.1, except for `django.contrib.postgres.field.JSONField`
